### ISSUE TITLE
Volumecontrol issue in Windows 10.

### ISSUE THREAD

**Issue by sm0yyq** [IngeHP/Sidplay3#6]
Hi, i have downloaded and tried out Sidplay3 on my Windows 10 machine and i got a problem.

I seem not be able to change the volumeslider for Sidplayer3 in the volumecontrol panel in Windows 10.
If i do degrease the volume it, changes it back to full volume directly.

Any ideas?

**Comment by IngeHP** [comment]
You are absolutely right. This is simply bad programming as the program doesn't care about the volume settings in the Windows environment. I will try to fix that.

Fra: sm0yyq ***@***.***>
Sendt: lørdag 31. desember 2022 11:17
Til: IngeHP/Sidplay3 ***@***.***>
Kopi: Subscribed ***@***.***>
Emne: [IngeHP/Sidplay3] Volumecontrol issue in Windows 10. (Issue #6)


Hi, i have downloaded and tried out Sidplay3 on my Windows 10 machine and i got a problem.

I seem not be able to change the volumeslider for Sidplayer3 in the volumecontrol panel in Windows 10.
If i do degrease the volume it, changes it back to full volume directly.

Any ideas?

-
